spring-swagger-codegen/openapi-custom-generator/pom.xml

```diff
@@ -1,17 +1,17 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
please remove identation of line 2 below

Reply:
i fixed it, thanks!